packages/material-ui/src/Toolbar/Toolbar.js

```diff
@@ -53,6 +62,11 @@ Toolbar.propTypes = {
    * @ignore
    */
   className: PropTypes.string,
+  /**
+   * The component used for the root node.
+   * Either a string to use a DOM element or a component.
+   */
+  component: PropTypes.elementType,
```

Comment:
Needs changelog attention

Reply:
As a new feature?

Reply:
@oliviertassinari Yes. Just in case someone is interested